apps/active_directory_ds.go

```diff
@@ -0,0 +1,26 @@
+package apps
```

Comment:
Nit: add copyright header (just copy from any of the other apps)

Reply:
done!